tests/api_tests/test_create_index.py

```diff
@@ -36,11 +36,14 @@ def test_simple_index_creation(self):
         expected_settings = {
```

Comment:
Do we need add_documents tests?

Reply:
We didn't change the api for add documents, and we cover add documents in the `marqo` repo. Do you think it's still necessary that we add languagebind add docs tests here as well? My concern is that it's a bit of a big model, and it would make these tests go slower

Reply:
I think current tests should cover the add docs api right?

Reply:
added